health-sdk/sdk/src/test/java/net/gini/android/health/sdk/moreinformation/MoreInformationTest.kt

```diff
@@ -0,0 +1,102 @@
+            PaymentProviderAppsState.Success(
+                listOf(
+                    PaymentProviderApp(
+                        name = "payment provider",
+                        icon = null,
+                        colors = PaymentProviderAppColors(-1, -1),
+                        paymentProvider = PaymentProvider(
+                            id = "payment provider id",
+                            name = "payment provider name",
+                            packageName = "com.paymentProvider.packageName",
+                            appVersion = "appVersion",
+                            colors = PaymentProvider.Colors(backgroundColorRGBHex = "", textColoRGBHex = ""),
+                            icon = ByteArray(0),
+                        )
+                    ),
+                    PaymentProviderApp(
+                        name = "payment provider 2",
+                        icon = null,
+                        colors = PaymentProviderAppColors(-1, -1),
+                        paymentProvider = PaymentProvider(
+                            id = "payment provider id",
+                            name = "payment provider name",
+                            packageName = "com.paymentProvider.packageName",
+                            appVersion = "appVersion",
+                            colors = PaymentProvider.Colors(backgroundColorRGBHex = "", textColoRGBHex = ""),
+                            icon = ByteArray(0),
+                        )
+                    ),
+                    PaymentProviderApp(
+                        name = "payment provider 3",
+                        icon = null,
+                        colors = PaymentProviderAppColors(-1, -1),
+                        paymentProvider = PaymentProvider(
+                            id = "payment provider id",
+                            name = "payment provider name",
+                            packageName = "com.paymentProvider.packageName",
+                            appVersion = "appVersion",
+                            colors = PaymentProvider.Colors(backgroundColorRGBHex = "", textColoRGBHex = ""),
+                            icon = ByteArray(0),
+                        )
+                    )
+                )
+            )
+        )
+
+        // When
+        launchFragmentInContainer {
+            MoreInformationFragment.newInstance(paymentComponent)
+        }
+
+        // Then
+        onView(withId(R.id.ghs_payment_providers_icons_list)).check { view, _ -> assertThat ((view as RecyclerView).adapter!!.itemCount).isEqualTo(3) }
+    }
+
+    @Test
+    fun `displays all FAQ questions`() {
+        // When
+        launchFragmentInContainer {
+            MoreInformationFragment.newInstance(paymentComponent)
+        }
+
+        // Then
+        onView(withId(R.id.ghs_faq_list)).check { view, _ -> assertThat ((view as ExpandableListView).adapter!!.count).isEqualTo(6) }
```

Comment:
[Suggestion] This test will fail if we add or remove questions. What do you think of making the list of FAQs internally accessible (and adding the `@VisibleForTesting` to make it explicit why it's not private) and then use the list's length in the assertion here?

Alternatively you could pass the faq list as an argument to the fragment's `newInstance()`, but that needs a bit bigger refactoring. No need to do that now because we might refactor this screen in the future to fetch the questions from our API. Until then we keep the questions hardcoded in the fragment.

Reply:
Thank you for the suggestions - I went with the first option since there is a possibility we'll need to refactor to get the FAQ's from the API.